scripts/minisat-2.2.1-patch

```diff
@@ -1,7 +1,8 @@
-diff -urN minisat-2.2.1/minisat/core/Solver.cc minisat-2.2.1.patched/minisat/core/Solver.cc
---- minisat-2.2.1/minisat/core/Solver.cc	2011-02-21 13:31:17.000000000 +0000
-+++ minisat-2.2.1.patched/minisat/core/Solver.cc	2016-03-05 16:21:17.000000000 +0000
-@@ -210,7 +210,7 @@
+diff --git a/minisat/core/Solver.cc b/minisat/core/Solver.cc
+index 501393d..b450b73 100644
+--- a/minisat/core/Solver.cc
++++ b/minisat/core/Solver.cc
+@@ -210,7 +210,7 @@ void Solver::cancelUntil(int level) {
```

Comment:
I'd prefer for those changes to not be included (and similarly below), but also I'm not going to embark on bike shedding here.